### PR TITLE
Enable pylint test

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #81

### CODE REVIEW

File: golem_messages/message/__init__.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# For backwards compatibility import all old messages by name
+# pylint: disable=unused-import
+# pylint: disable=cyclic-import
+from golem_messages.message.base import Message  # noqa
```
Comment (maaktweluit) [inline golem_messages/message/__init__.py]: maybe all those `# noqa` blocks can also be a `pylint: disable=` before the block

File: golem_messages/message/__init__.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
+# For backwards compatibility import all old messages by name
+# pylint: disable=unused-import
+# pylint: disable=cyclic-import
```
Comment (maaktweluit) [inline golem_messages/message/__init__.py]: should we enable these again after this block?

File: golem_messages/message/base.py [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+import time
+
+import golem_messages
+
+from golem_messages import datastructures
+from golem_messages import exceptions
+from golem_messages import serializer
+from golem_messages import settings
+
+logger = logging.getLogger('golem.network.transport.message')
```
Comment (maaktweluit) [inline golem_messages/message/base.py]: no default `__name__` for all loggers?

File: golem_messages/message/base.py [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+    delta = now - msgdt
+    delta_future = msgdt - now
+    logger.debug('msgdt %s Δ %s Δfuture %s', msgdt, delta, delta_future)
+    if delta > settings.MSG_TTL:
+        raise exceptions.MessageTooOldError()
+    if delta_future > settings.FUTURE_TIME_TOLERANCE:
+        raise exceptions.MessageFromFutureError()
+
+
+def _fake_sign(s):  # pylint: disable=unused-argument
```
Comment (maaktweluit) [inline golem_messages/message/base.py]: unused argument can be replaced with `_*`

File: golem_messages/message/base.py [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+
+    HDR_LEN = 11
+    SIG_LEN = 65
+    PAYLOAD_IDX = HDR_LEN + SIG_LEN
+
+    TYPE = None
+    ENCRYPT = True
+    ENUM_SLOTS = {}
+
+    def __init__(self, timestamp=None, encrypted=False, sig=None,  # noqa TODO #88 pylint: disable=too-many-arguments
```
Comment (maaktweluit) [inline golem_messages/message/base.py]: nice todo!

File: golem_messages/message/__init__.py (hunk 2 above)
Comment (jiivan) [reply golem_messages/message/__init__.py]: done

File: golem_messages/message/__init__.py (hunk 1 above)
Comment (jiivan) [reply golem_messages/message/__init__.py]: if `pycodestyle` used block I would prefer block :)

File: golem_messages/message/base.py (hunk 3 above)
Comment (jiivan) [reply golem_messages/message/base.py]: It was left like this for backwards compatibility (before moving `golem.network.transport.message` to `golem_messages.message` and then `golem_messages.message.base`)

File: golem_messages/message/base.py (hunk 4 above)
Comment (jiivan) [reply golem_messages/message/base.py]: fixed

File: golem_messages/message/__init__.py (hunk 1 above)
Comment (maaktweluit) [reply golem_messages/message/__init__.py]: damn you `pycodestyle`! ;)

**Pull request merged**